Lib/pathlib.py

```diff
@@ -164,30 +164,15 @@ class _RecursiveWildcardSelector(_Selector):
     def __init__(self, pat, child_parts, flavour, case_sensitive):
         _Selector.__init__(self, child_parts, flavour, case_sensitive)
 
-    def _iterate_directories(self, parent_path, scandir):
+    def _iterate_directories(self, parent_path):
```

Comment:
What's up with performance as a result of this change? I feel like it might get slower. Does it?

Reply:
Overall it's about 5% slower when expanding `**` wildcards.

I don't think it matters too much. In #104244 I made the same case about 35% faster, and there's more speedups to come.